internal/backends/winit/lib.rs

```diff
@@ -235,8 +235,16 @@ impl BackendBuilder {
     /// slint::platform::set_platform(Box::new(backend));
     /// ```
     pub fn build(self) -> Result<Backend, PlatformError> {
-        let event_loop_builder =
-            self.event_loop_builder.unwrap_or_else(winit::event_loop::EventLoop::with_user_event);
+        let event_loop_builder = self.event_loop_builder.unwrap_or_else(|| {
+            #[allow(unused_mut)]
+            let mut default_builder = winit::event_loop::EventLoop::with_user_event();
+            #[cfg(all(feature = "muda", target_os = "macos"))]
+            {
+                use winit::platform::macos::EventLoopBuilderExtMacOS;
+                default_builder.with_default_menu(false);
+            }
```

Comment:
Personally, i'd do

```suggestion
            #[cfg(all(feature = "muda", target_os = "macos"))]
            winit::platform::macos::EventLoopBuilderExtMacOS::with_default_menu(default_builder, false);
```

But i guess that's just a question of style to get less lines. Either way is fine.

Reply:
Good idea, that's indeed nicer